app/Resources/views/admin/blog/index.html.twig

```diff
@@ -5,6 +5,8 @@
 {% block main %}
     <h1>{{ 'title.post_list'|trans }}</h1>
 
+    {{ helper.render_flash_messages() }}
```

Comment:
Rather than using a macro, I suggest using an include of a template rendering flash messages